app/views/static_pages/home/_success_stories.html.erb

```diff
@@ -12,7 +12,7 @@
         </div>
 
         <div class="flex-1">
-            <p><%= link_to story.student_name, story.social_media_link, class: 'text-gold-600' %></p>
+            <p><%= link_to story.student_name, story.social_media_link, class: 'underline hover:no-underline text-gray-600 hover:text-gray-800 dark:text-gray-400 dark:hover:text-gray-300' %></p>
```

Comment:
Nit: Similar to the announcement link, I think we can make this stand out more. It's pretty much the heading of the card:

```suggestion
            <p><%= link_to story.student_name, story.social_media_link, class: 'underline hover:no-underline text-gray-900 hover:text-gray-700 dark:text-gray-200 dark:hover:text-gray-300' %></p>
```